docs/cn/about-appium/intro.md

```diff
@@ -60,7 +60,7 @@ Appium 的 `beta` 版本可以通过 NPM 使用 `npm install -g appium@beta` 指
 
 ### 入门指南
 
-恭喜！你现在有足够的知识来使用 Appium 了。 为什么不前往 [入门指南](/docs/cn/about-appium/getting-started.md) 了解更多详细的要求和指南呢？
+恭喜！你现在有足够的知识来使用 Appium 了。 为什么不前往 [入门指南](/docs/en/about-appium/getting-started.md)(English) 了解更多详细的要求和指南呢？
```

Comment:
is there any reason to have links to English docs in Chinese?

Reply:
We do not have proper Chinese translated files, and I don't know Chinese well...  (I only can remove them :P)
So, simply I relaced the missing resource links in `cn` with the existing resource, in English one for now

Reply:
I'd rather keep some universal placeholder there, like `help us with the translation`. My point is that when a person is checking the docs in Chinese then there is quite low probability that he or she would understand the English one being unexpectedly redirected there.

Reply:
Ah, got it. I'd love to do such things in another PR since maybe we should clean up current `cn` translations more.

Current appiumio hasn't been updated since late last year. So, would like to fix the CI cycle for English for now. Once the cycle will get back to normal, would like to continue to work more.